sdk/python/lib/pulumi/runtime/rpc.py

```diff
@@ -43,6 +44,14 @@
 _special_secret_sig = "1b47061264138c4ac30d75fd1eb44270"
 """special_secret_sig is a randomly assigned hash used to identify secrets in maps. See pkg/resource/properties.go"""
 
+_INT_OR_FLOAT = six.integer_types + (float,)
+
+def isLegalProtobufValue(value: Any) -> bool:
+    """
+    Returns True if the given value is a legal Protobuf value.
+    """
+    return value is None or isinstance(value, (bool, six.string_types, _INT_OR_FLOAT, dict, list))
```

Comment:
Is this documented or discoverable anywhere we can reference here?

Reply:
We could link to the source: https://github.com/protocolbuffers/protobuf/blob/master/python/google/protobuf/internal/well_known_types.py#L714-L732

As far as I can tell, [the docs](https://developers.google.com/protocol-buffers/docs/reference/python-generated#struct) don't mention this at all.

Reply:
An internal facing comment linking to the source wouldn't hurt.

Reply:
Done